examples/lws/server_lws_http_client.c

```diff
@@ -0,0 +1,48 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+
+#include <open62541/plugin/eventloop.h>
+#include <open62541/plugin/log_stdout.h>
+
+UA_Boolean running = true;
+static UA_EventLoop *el;
+
+static void stopHandler(int sign) {
+    UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_CLIENT, "Received Ctrl-C");
+    running = false;
+}
+
+int main(int argc, const char **argv) {
+    signal(SIGINT, stopHandler);
+    signal(SIGTERM, stopHandler);
+
+    UA_ConnectionManager *cm = UA_ConnectionManager_new_HTTP(UA_STRING("httpCM"));
+    el = UA_EventLoop_new_POSIX(UA_Log_Stdout);
+    el->registerEventSource(el, &cm->eventSource);
+    el->start(el);
+
+    UA_String url = UA_STRING("api.open-meteo.com");
+    UA_UInt16 port = 443;
+    UA_String path = UA_STRING("/v1/forecast?latitude=49.01&longitude=8.40&hourly=temperature_2m");
+    UA_String method = UA_STRING("GET");
+    size_t paramsSize = 4;
+    UA_KeyValuePair params[paramsSize];
+    params[0].key = UA_QUALIFIEDNAME(0, "url");
+    UA_Variant_setScalar(&params[0].value, &url, &UA_TYPES[UA_TYPES_STRING]);
+    params[1].key = UA_QUALIFIEDNAME(0, "port");
+    UA_Variant_setScalar(&params[1].value, &port, &UA_TYPES[UA_TYPES_UINT16]);
+    params[2].key = UA_QUALIFIEDNAME(0, "path");
+    UA_Variant_setScalar(&params[2].value, &path, &UA_TYPES[UA_TYPES_STRING]);
+    params[3].key = UA_QUALIFIEDNAME(0, "method");
+    UA_Variant_setScalar(&params[3].value, &method, &UA_TYPES[UA_TYPES_STRING]);
+
+    UA_KeyValueMap map = {paramsSize, params};
+
+    cm->openConnection(cm, NULL, NULL, NULL, NULL);
```

Comment:
Add a callback to log the received result.
Also add the handling of failure cases where the remote side does not exist.

How are HTTP status codes lik 404 handled?
Given to the callback as part of the parameter map?